core/storage/config/gae_models_test.py

```diff
@@ -194,3 +210,21 @@ def test_commit_with_updated_rules(self) -> None:
         assert retrieved_model is not None
 
         self.assertEqual(retrieved_model.rules, new_rules)
+
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            config_models.PlatformParameterModel.
+            get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
+        )
+
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.VersionedModel.get_export_policy()
+        sample_dict.update({
+            'rules': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'rule_schema_version': base_models.EXPORT_POLICY.NOT_APPLICABLE
+        })
+        self.assertEqual(
+            config_models.PlatformParameterModel.get_export_policy(),
+            sample_dict
+        )
```

Comment:
ditto: Too similar to the implementation of the code being tested

Reply:
Done!